src/MusicalScore/Graphical/DrawingEnums.ts

```diff
@@ -1,3 +1,5 @@
+import Dictionary from "typescript-collections/dist/lib/Dictionary";
```

Comment:
We might want to use
```ts
import * as Collections from "typescript-collections";
...
new Collections.Dictionary<...>();
```

instead, as this is the preferred usage according to https://github.com/basarat/typescript-collections#usage.
If they change the directory structure of their bundle, we're doomed with that path-dependent imports :D

Reply:
Damn Visual Studio Code autoimporter. So damn convenient and such a b*** :)